File: BaekJoon/Array/4344.py
import sys

c = int(input())

for _ in range(c):
    arr = list(map(int, sys.stdin.readline().split()))
    arr = arr [1:]
    avg = sum(arr)/len(arr)
    upper = [ x for x in arr if x > avg]
    rate = float(len(upper)) / len(arr) *100
    print("{:.3f}%".format(rate))

File: BaekJoonSwift/BaekJoonSwift/Array/4344.swift
import Foundation

let n = Int(readLine()!)!

for _ in 0..<n {
    var iValues = readLine()!.split(separator: " ").map { Int($0)! }
    let len = iValues[0]
    let arr = iValues[1...]
    let sum = arr.reduce(0, {$0+$1})
    let avg = Double(sum) / Double(len)
    let upperRate = Double(arr.filter{Double($0) > avg}.count)/Double(len)
    
    print("\(String(format: "%.3f", upperRate*100))%")
}
